lib/erlef/publications.ex

```diff
@@ -1,9 +0,0 @@
-defmodule Erlef.Publications do
```

Comment:
I'm interested as to why you think this should be deleted?

Reply:
It is has been replaced by Erlef.AcademicPapers, to be more clear and more adaptative to the Phoenix convention(when saying publications, it can be any type of publications, and we don't want to put all publications in the same module)

Reply:
That definitely depends 😁 A context is a fine place for being generic, in fact, what call out was intentional. A generic context for publications (e.g., books, academic papers, etc.) 

 It's when a context has the possibility of getting out of hand and encapsulating too much behavior that we need a new context. 

This is documented roughly the same way [here](https://hexdocs.pm/phoenix/contexts.html). 

Let me pause there to get more of thoughts after digesting this. What's more I will refrain from further review until this conversation is resolved, as the decision we make here greatly influences the rest of the review 😄

Reply:
indeed, I rethinked more than once when I decided to rename publications, but finally I found that's academic papers management worths its own context... but this is nothing but just naming problem, I can restore publications if you see that it's better

Reply:
Let me think on this....

Reply:
Of course, take all your time 🙂